dominion/dominion-frontend/README.md

```diff
@@ -0,0 +1,29 @@
+## 🚀 Quick start: Dominion React Front End
```

Comment:
If relevant, this should be move`HURUmap-apps/README.md` or at least linked from there if too long/different.